vendor/windows_game_input/windows_game_input.odin

```diff
@@ -0,0 +1,1129 @@
+#+build windows
+/*
+	Bindings for Windows Game Input GDK:
+	https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/input/overviews/input-overview
+
+	Windows SDK 10.0.26100.0 is at least required to link with.
+*/
+package windows_game_input
+
+foreign import wgi {
+	"system:gameinput.lib",
+}
+
+import "core:c"
+import "core:sys/windows"
+
+// Enums
+GameInputArcadeStickButtons_Flag :: enum c.int {
```

Comment:
If everything is prefixed with `GameInput`, why not just remove the common prefix entirely?